### PR TITLE
feat: ux fixes

### PR DESCRIPTION
**Pull request by simboonlong**
- wasn't able to reach back to other tools, other than clicking home logo on top left
- we can put `wrap` and `verify` at top nav bar, but seeing since the tools are listed in a section-like area, use breadcrumbs within that section instead
- cleanup, prep images

### CODE REVIEW

**Comment by simboonlong** [comment]
im guessing i no longer has access to circleci, hence build error

File: .circleci/config.yml [hunk 1]
```diff
@@ -14,9 +14,9 @@ jobs:
       - run:
           name: lint
           command: npm run lint
-#      - run:
-#          name: test
```
Comment (Nebulis) [inline .circleci/config.yml:18]: Looks like ci is failing because the file is not valid

File: src/Router.tsx [hunk 2]
```diff
@@ -1,23 +1,58 @@
...
+            </Route>
+            <Route exact path="/verify">
+              <Verify />
+            </Route>
+            <Route path="/">
+              <Home />
+            </Route>
+          </Switch>
+        </section>
+      </main>
```
Comment (Nebulis) [inline src/Router.tsx:55]: Why not moving the footer too?

File: .circleci/config.yml (hunk 1 above)
Comment (simboonlong) [reply .circleci/config.yml:18]: orps indentation, ok thanks

File: src/Router.tsx (hunk 2 above)
Comment (simboonlong) [reply src/Router.tsx:55]: sure

**Review by Nebulis — APPROVED** [review]

**Pull request merged**